testing/perl-db_file-lock/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Automatically generated by apkbuild-cpan, template 2
+# Contributor: Timothy Legge <timlegge@gmail.com>
+# Maintainer: Timothy Legge <timlegge@gmail.com>
+pkgname=perl-db_file-lock
+_pkgreal=DB_File-Lock
+pkgver=0.05
+pkgrel=0
+pkgdesc="unknown"
```

Comment:
I think we need a better description what this is

Reply:
apkbuild-cpan grabs the description from the perl module abstract.  I thought that was unknown at the time but is likely a bug I have updated it manually and will look at apkbuild-cpan